### PR TITLE
Remove legacy JavaScript: cShowHide function

### PR DESCRIPTION
**Pull request by cfarm**
Removes unnecessary JS from [Find a Housing Counselor page](https://www.consumerfinance.gov/find-a-housing-counselor/?zipcode=05201). This JS was used for a toggle show-hide function for the Paperwork Reduction Act statement at the bottom of the list of results.

## Removals

- function from legacy JS file
- classes and html elements targeted by this JS 

## Changes

- added h3 heading to Paperwork Reduction Act statement text

## Testing

1. `./setup.sh`
1. `./runserver.sh`
1. Go to http://localhost:8000/find-a-housing-counselor/?zipcode=05201 and confirm the Paperwork Reduction act statement text is visible by default both with and without JavaScript on.

### Before
screen shot 2017-11-09 at 1 52 01 pm

### After
screen shot 2017-11-09 at 1 52 08 pm

## Notes

- Platform team issue #2327 https://GHE/CFGOV/platform/issues/2327 described this function as also powering the toggle on the español homepage but that does not appear to be this case. I will address removing that in a separate PR.
- This same JavaScript appears to be duplicated in nemo code ([see search results showing this function here](https://github.com/cfpb/cfgov-refresh/search?utf8=%E2%9C%93&q=cShowHide&type=)), @sebworks or @mistergone can you confirm this is as expected? Do I need to delete that too, or should I leave that to someone else who is working to kill WordPress..?

## Todos

-

## Checklist

* [ ] PR has an informative and human-readable title
* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [development playbook](https://github.com/cfpb/development)
* [ ] Passes all existing automated tests
* [ ] Any *change* in functionality is tested
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated
* [ ] Reviewers requested with the [Reviewer tool](https://help.github.com/articles/about-pull-request-reviews/) :arrow_right:

### CODE REVIEW

File: cfgov/legacy/templates/hud/housing_counselor_pdf_selfcontained.html [hunk 1]
```diff
@@ -252,7 +252,7 @@ <h2 class="cfpb_visually_hide">Agency</h2>
 															<span class="show-hide-name">View</span>
 															languages offered at this location
                             </button>
-													<div class="show-hide-content hud_hca_api_print_show">
+													<div class=" hud_hca_api_print_show">
```
Comment (sebworks) [inline cfgov/legacy/templates/hud/housing_counselor_pdf_selfcontained.html]: drop the space here

**Comment by sebworks** [comment]
looks good, 👍

**Comment by anselmbradford** [comment]
> This same JavaScript appears to be duplicated in nemo code

Similar to this, the associated styles to this are in e.g. https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/legacy/static/nemo/_/c/styles.css#L1959 and https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/v1/static-legacy/css/styles.css#L1952 can those be deleted too?

**Comment by cfarm** [comment]
I removed the relevant CSS too, didn't touch the `nemo` folder stuff because I'm not sure where else that could be used.

**Pull request merged**